cmd/gen.go

```diff
@@ -58,9 +58,9 @@ func init() {
 
 	// set flags of mtar command
 	mtarCmd.Flags().StringVarP(&mtarCmdSrc, "source", "s", "",
-		"The path to the MTA project; the current path is set as the default")
+		"The path to the MTA project; the current path is set as default")
 	mtarCmd.Flags().StringVarP(&mtarCmdTrg, "target", "t", "",
-		"The path to the results folder; the current path is set as the default")
+		`The path to the generated MTAR file; the path to the "mta_archives" subfolder of the current folder is set as default`)
```

Comment:
Same - isn't it the path to the folder in which the MTAR file is created?

Reply:
changed